packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -0,0 +1,132 @@
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * SIZES                                                                                           *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-RadioCardGroupItem {
+  padding: var(--radio-card-group-item-padding);
+  border-radius: var(--radio-card-group-item-radius);
+  gap: var(--radio-card-group-item-gap);
+}
+
+@breakpoints {
+  .rt-RadioCardGroupRoot {
+    &:where(.rt-r-size-1) {
+      --radio-card-group-item-padding: var(--space-3);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-2);
+    }
+    &:where(.rt-r-size-2) {
+      --radio-card-group-item-padding: var(--space-4);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-3);
+    }
+    &:where(.rt-r-size-3) {
+      --radio-card-group-item-padding: var(--space-5);
+      --radio-card-group-item-radius: var(--radius-5);
+      --radio-card-group-item-gap: var(--space-4);
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * VARIANTS                                                                                        *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+/* surface */
+
+.rt-RadioCardGroupRoot:where(.rt-variant-surface) {
+  :where(.rt-RadioCardGroupItem) {
+    background-color: var(--color-surface);
+    box-shadow: inset 0 0 0 1px var(--gray-a5);
+
+    &:where(:hover) {
+      box-shadow: inset 0 0 0 1px var(--gray-a7);
+    }
+
+    &:where([data-state='checked']) {
+      outline: 2px solid var(--accent-9);
+      outline-offset: -1px;
+    }
+
+    &:where(:focus-visible) {
+      outline: 2px solid var(--accent-9);
+      outline-offset: -1px;
+      box-shadow: 0 0 0 4px var(--accent-a6);
+    }
+  }
+
+  &:not(:where(.rt-high-contrast)) :where(.rt-RadioCardGroupItem):where([data-state='checked']) {
+    background-color: var(--accent-a2);
+    color: var(--accent-12);
+  }
+  &:where(.rt-high-contrast) :where(.rt-RadioCardGroupItem) {
+    &:where([data-state='checked']) {
+      outline-color: var(--accent-12);
+    }
+    &:where(:focus-visible) {
+      outline-color: var(--accent-12);
+    }
+  }
+}
+
+/* classic */
+
+.rt-RadioCardGroupRoot:where(.rt-variant-classic) {
+  :where(.rt-RadioCardGroupItem) {
+    background-color: var(--color-panel);
```

Comment:
Not sure what the background should be for classic, should it remain `--color-surface` and just differ by the shadow?